FirebaseMessaging/Sources/FIRMessagingUtilities.m

```diff
@@ -89,6 +95,25 @@ int64_t FIRMessagingCurrentTimestampInMilliseconds(void) {
   return [FIROptions defaultOptions].googleAppID;
 }
 
+BOOL FIRMessagingIsWatchKitExtension(void) {
```

Comment:
would this be able to differentiate from old version because it would be nice to keep the older version working as well.

Also just checking to see if you have tested on real watch yet? If not, I can help test it on my device.